.circleci/documentation-versions.py

```diff
@@ -19,20 +19,22 @@ def main():
     except IndexError:
         sys.exit("Missing build dir path: python /path/docs")
 
-    ignored_dirs = {".git"}
+    keep_versions = ['1.5.0']
+    ignored_dirs = {".git", *keep_versions}
     dirs = [
         item
         for item in os.scandir(build_dir)
         if item.is_dir() and item.name not in ignored_dirs
     ]
     sorted_dirs = sorted(dirs, reverse=True, key=attrgetter("name"))
-    active_versions = [item.name for item in sorted_dirs[:7]]
+    active_versions = [item.name for item in sorted_dirs[:7]] + keep_versions
```

Comment:
what is the importance of `:7`? maybe we could put that into a constant?

Reply:
Sounds like a good idea, it makes sure that last `n-1` versions are displayed and old versions removed, as we discussed before and chosen number of 6.

then it builds a `version-helper.js` which fills in this 
![image](https://user-images.githubusercontent.com/7480694/65697422-2f428a80-e07b-11e9-9e84-3fb1df442ad1.png)

+ outputs the versions to be removed (which reminds me that I need to update the code in PR as it's not exactly right)